clustered/integration-test/src/test/java/org/ehcache/clustered/management/EhcacheConfigWithManagementTest.java

```diff
@@ -50,9 +50,11 @@
       + "</ohr:offheap-resources>" +
       "</config>\n";
 
-  private static final List<File> MANAGEMENT_PLUGINS = Stream.of(System.getProperty("managementPlugins", "").split(File.pathSeparator))
-    .map(File::new)
-    .collect(Collectors.toList());
+  private static final List<File> MANAGEMENT_PLUGINS = System.getProperty("managementPlugins") == null ?
+    Collections.emptyList() :
+    Stream.of(System.getProperty("managementPlugins").split(File.pathSeparator))
```

Comment:
Goal now is to put the serevr libs directly into the kit after it has been unzipped, so that ALL integration tests are using now the management stack.

Before, we had to pass a system property and add the server libs in only specifically some integration tests (m&m ones). System property "managementPlugins" is not there anymore so `Collections.emptyList()` will be returned.

I could remove this dead code though...